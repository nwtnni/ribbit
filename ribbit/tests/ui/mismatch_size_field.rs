#[derive(ribbit::Pack, Copy, Clone)]
#[ribbit(size = 16)]
struct A {
    #[ribbit(size = 3)]
    lo: u8,

    #[ribbit(size = 13)]
    hi: u8,
}

fn main() {}
