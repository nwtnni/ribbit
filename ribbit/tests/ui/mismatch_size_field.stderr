error: Size attribute #[ribbit(size = 3)] does not match size of u8: 8
 --> tests/ui/mismatch_size_field.rs:4:9
  |
4 |     lo: u8,
  |         ^^
